future_challenges_for_RSEng.tex

```diff
@@ -550,11 +550,15 @@ \subsection{How have academic Institutions transformed by then?}
 We also hope that specialised clusters form, which pool the skills of certain specialised RSEs with certain RSE units specializing in certain tasks like data management, HPC, or visualisation.
 At universities which get such RSE units, we want to see that researchers naturally accept them as part of their available services.
 Additionally, researchers will seamlessly interact with infrastructures for using research software.
+Replacement of aging staff has led to the creation of more RSE professorships.
 \end{whatis}
 Academia, while claiming to be innovative~\cite{wzvgbayern},
 has organisationally often adopted time scales for change that are more reminiscent
-to institutions thinking on the time scale of generations.
-Nevertheless, the presence, even today, of initiatives like \emph{the Hidden Ref}~\cite{hiddenref}
+to institutions thinking on the time scale of societal generations.
+While sounding negative, this can also be an opportunity,
+if in the time-frame till 2035 a large number of aging, permanent staff is replaced by new people that have observed the
+digital transformation first-hand and have actively participated in it through various grassroots initiatives.\\
+On the other hand, the presence, even today, of initiatives like \emph{the Hidden Ref}~\cite{hiddenref}
```

Comment:
Suggesting a rewording here - this is personal preference but I also think the "is" on L559 should be "are"?
```suggestion
if in the time-frame of the next decade, a large number of permanent staff reach retirement and are replaced by a new generation of researchers and technical professionals who have seen the
digital transformation first-hand and have actively participated in it through various grassroots initiatives.\\
On the other hand, the presence, even today, of initiatives like \emph{the Hidden Ref}~\cite{hiddenref}
```